core/vnl/vnl_vector_ref.h

```diff
@@ -24,28 +24,20 @@ template <class T>
 class VNL_EXPORT vnl_vector_ref : public vnl_vector<T>
 {
  public:
-  typedef vnl_vector<T> Base;
+  using Base = vnl_vector<T>;
 
   //: Constructor
   // Do \e not call anything else than the default constructor of vnl_vector<T>
-  vnl_vector_ref(unsigned n, T *space) : vnl_vector<T>() {
-    Base::data = space;
-    Base::num_elmts = n;
-  }
+  vnl_vector_ref(unsigned n, T *space);
 
   //: Copy constructor
   // Do \e not call anything else than the default constructor of vnl_vector<T>
   // (That is why the default copy constructor is \e not good.)
```

Comment:
What does this backslash-e do in the comment? And it appears on the previous line too.